en/contributing/documentation.rst

```diff
@@ -72,26 +73,32 @@ preceded and followed by a blank line.
 Paragraphs
 ----------
 
-Paragraphs are simply blocks of text, with all the lines at the same level of indentation.
-Paragraphs should be separated by more than one empty line.
+Paragraphs are simply blocks of text, with all the lines at the same level of
+indentation. Paragraphs should be separated by more than one empty line.
 
 Inline Markup
 -------------
 
 * One asterisk: *text* for emphasis (italics)
+  We'll use it for Object names, excluding the following word "object",
+  for method/function names -- include "()".
 
-  * ``*text*``
+  * ``*PagesController*``, ``**config()**``, etc.
```

Comment:
I can move them to inline code.
What sort of things could be in italics ? Currently, I could find general emphasis, some words or portions of sentences.